src/mc_tools.jl

```diff
@@ -54,65 +51,63 @@ function Base.show(io::IO, mc::MarkovChain)
     println(io, mc.p)
 end
 
+# solve x(P-I)=0 by eigen decomposition
 function eigen_solve{T}(p::Matrix{T})
     ef = eigfact(p')
-    isunit = map(x->isapprox(x, 1), ef.values)
+    isunit = map(x->isapprox(x,1), ef.values)
     x = real(ef.vectors[:, isunit])
-    x ./= sum(x, 1) # normalisation
+    x ./= sum(x,1) # normalisation
```

Comment:
To follow the style of the rest of the library, a comma must _always_ be followed by a space.

This applies multiple times in this PR, but I will only mention it here to preserve everyone's inbox!